app/templates/macros/nav_macros.html

```diff
@@ -28,26 +28,25 @@
     {% set calendar = [] %}
     {% set resources = [] %}
     {% set scattergram = [] %}
+    {% set colleges = [] %}
     {% if current_user.is_authenticated %}
       {% set user = ([(current_user.role.index + '.index', current_user.role.name + ' Dashboard', 'user')]) %}
       {% set calendar = ([('main.calendar', 'Calendar', 'info')]) %}
       {% set scattergram = ([('main.scattergram', 'Scattegram', 'info')]) %}
       {% set resources = ([('main.resources', 'Resources', 'info')]) %}
+      {% if current_user.is_admin() or current_user.is_counselor() %}
+        {% set calendar = ([('counselor.calendar', 'Calendar', 'info')]) %}
+        {% set colleges = ([('counselor.colleges', 'Colleges', 'info')]) %}
+      {% endif %}
     {% endif %}
-    {{ render_menu_items( endpoints +  user + calendar + resources + scattergram) }}
+    {{ render_menu_items( endpoints +  user + calendar + resources + scattergram + colleges) }}
 {% endmacro %}
 
 {# This renders the right hand side of the navigation bar. If the user is logged in, it links to
    manage their account and logout (account routes). Otherwise, it links to register and login.
 #}
 {% macro account_items(current_user) %}
-    {% if current_user.is_authenticated %}
-      {% set endpoints = [
-        ('account.manage', 'Your Account', 'settings'),
-        ('account.logout', 'Log out', 'sign out')
-      ] %}
-      {{ render_menu_items(endpoints) }}
-    {% else %}
+    {% if not current_user.is_authenticated %}
       {% set endpoints = [
```

Comment:
Before, we check if the user is authenticated in this file, and render the appropriate right-hand nav buttons. However, since the `render_nav` function takes an optional `dropdown` argument, I've refactored this so that we only render these account buttons if no user is logged in; on the other hand if a user _is_ logged in, we simply pass on the required account buttons to `render_nav`.